.pre-commit-config.yaml

```diff
@@ -35,3 +36,8 @@ repos:
         additional_dependencies: [types-all]
         args: [--explicit-package-bases, --ignore-missing-imports]
         exclude: ^dynamic_importer/samples/|^files/
+-   repo: https://github.com/leoll2/copyright_notice_precommit
+    rev: 0.1.1
+    hooks:
+    -   id: copyright-notice
```

Comment:
Do we want to drop this into conserv too?

Reply:
Totally can. I was just testing here to see how it works.